packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewer.tsx

```diff
@@ -31,6 +31,11 @@ type Props = PdfDisplayProps & {
    */
   textLayerClassName?: string;
 
+  /**
+   * disable textLayer if true
```

Comment:
Similar comments to the component above (sentence case, default value). But fine to keep this named `disableTextLayer`, since adding `Pdf` in there would be redundant for `PdfViewer` component.